pkg/parser/getters/getters.go

```diff
@@ -46,3 +48,9 @@ type IPGetter interface {
 	// GetIPIdentity fetches information known about a remote IP.
 	GetIPIdentity(ip net.IP) (identity ipcache.IPIdentity, ok bool)
 }
+
+// ServiceGetter fetches service metadata.
+type ServiceGetter interface {
+	GetServiceByAddr(ip net.IP, port int) (service pb.Service, ok bool)
+	GetServiceByID(id int64) (service pb.Service, ok bool)
```

Comment:
nit: This function seems unused - did you have a future use for in mind?

Reply:
Not really. I thought I would need to use it at some point but it turns out than it wasn't necessary and I just left the code there as it was already implemented/tested.
I think that in a way, it makes it clear that you can retrieve a service in the cache either by address or by ID but if you prefer, I have no strong feelings about removing it.

Reply:
Sounds good, let's leave it in. :+1: